Philpott.jl

```diff
@@ -0,0 +1,14 @@
+srand(1111)
```

Comment:
Maybe move this file in `/run` folder?

Reply:
We can. Anyway this file is not destined to stay.